src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/FileTabCharacterCheck.java

```diff
@@ -68,6 +68,18 @@
  *   &lt;property name=&quot;eachLine&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>violation</p>
+ * <pre>
+ * for(int i=0; i&lt;10; i++){
+ *     //violation(extra spaces after open bracket)
+ * }
+ * </pre>
+ * <p>acceptable code</p>
+ * <pre>
+ * for(int i=0; i&lt;10; i++){
+ *     //OK(no space after bracket)
```

Comment:
this comment seems to refer to a different check (spaces after open bracket) and not tab characters